### PR TITLE
Fix dumpall primary key name

### PR DESCRIPTION
**Pull request by nvcnvn**

### CODE REVIEW

**Comment by nvcnvn** [comment]
This is a mistake, I want to create a PR to my own fork, sorry for the noise!

**Comment by polarweasel** [comment]
@nvcnvn was reopening this issue on purpose, or by accident? If on purpose, how can we help?

**Comment by nvcnvn** [comment]
> @nvcnvn was reopening this issue on purpose, or by accident? If on purpose, how can we help?

sorry, I'm trying to fix this https://github.com/yugabyte/yugabyte-db/issues/4806 I mean to open back the PR and wait for the CI to finished but totally forget.

**Comment by emhna** [comment]
Thanks @nvcnvn. The code looks good. We will verify it in our internal test framework to confirm.

File: src/postgres/src/test/regress/expected/yb_ysql_dump.out [hunk 1]
```diff
@@ -38,7 +38,7 @@ ALTER TABLE public.chat_user OWNER TO yugabyte_test;
 CREATE TABLE public.rls_private (
     k integer NOT NULL,
     v text,
-    PRIMARY KEY((k) HASH)
+    CONSTRAINT rls_private_pkey PRIMARY KEY((k) HASH)
```
Comment (emhna) [inline src/postgres/src/test/regress/expected/yb_ysql_dump.out:41]: Please ensure to add constraint for `chatID` on line 28 above as well eg.`CONSTRAINT chat_user_pkey PRIMARY KEY(("chatID") HASH)`

**Review by emhna — COMMENTED** [review]
Please ensure to add constraint for `chatID` as well eg.`CONSTRAINT chat_user_pkey PRIMARY KEY(("chatID") HASH)`.

File: src/postgres/src/test/regress/expected/yb_ysql_dump.out (hunk 1 above)
Comment (nvcnvn) [reply src/postgres/src/test/regress/expected/yb_ysql_dump.out:41]: Done!

**Review by emhna — CHANGES_REQUESTED** [review]
@nvcnvn, could you please ensure to rebase with the latest changes in the master branch? When I get the [patch](https://patch-diff.githubusercontent.com/raw/yugabyte/yugabyte-db/pull/5667.patch) of your diff, it's showing an old change in the `TestYsqlDump.java` file for line 168.
This is the [current code](https://github.com/yugabyte/yugabyte-db/blob/d28d82e8c48bb068b538a3ea0e5165525e46838d/java/yb-pgsql/src/test/java/org/yb/pgsql/TestYsqlDump.java#L168) for verifying the dumped output.

**Comment by nvcnvn** [comment]
> @nvcnvn, could you please ensure to rebase with the latest changes in the master branch? When I get the [patch](https://patch-diff.githubusercontent.com/raw/yugabyte/yugabyte-db/pull/5667.patch) of your diff, it's showing an old change in the `TestYsqlDump.java` file for line 168.
> This is the [current code](https://github.com/yugabyte/yugabyte-db/blob/d28d82e8c48bb068b538a3ea0e5165525e46838d/java/yb-pgsql/src/test/java/org/yb/pgsql/TestYsqlDump.java#L168) for verifying the dumped output.

Hi @emhna I messed around with the rebase, strange that the update not show in the github compare view.
The TestYsqlDump.java have swapped between expect and actually in assert, Its make  a bit of confusing for me so I swapped them back.

**Review by emhna — APPROVED** [review]

**Pull request merged**